scripts/generators/stubGenerator.sh

```diff
@@ -0,0 +1,333 @@
+# This script generates a stub from a given interface
```

Comment:
What works or not depends on the shell used. Add a shebang. (#!/bin/bash or #!/bin/sh)